src/ProductAnalytics/Context.tsx

```diff
@@ -18,13 +18,15 @@ export type ProductAnalyticsContextState = {
     value?: number,
   ) => Promise<void>
   trackScreenView: (screen: string) => Promise<void>
+  resetProductAnaltyics: () => Promise<void>
```

Comment:
current naming certainly works, but what do you think about calling this something like `resetUserConsent` instead?